demo/KevinDemo/Stories/AccountLinking/AccountLinkingView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AccountLinkingView.swift
+//  KevinDemo
+//
+//  Created by Daniel Klinge on 10/06/2022.
+//
+
+import SwiftUI
+
+public struct AccountLinkingView: View {
+    
+    @StateObject var viewModel = AccountLinkingViewModel()
+    
+    public var body: some View {
+        NavigationView {
+            ZStack {
+                Color("PrimaryBackgroundColor").ignoresSafeArea()
+
+                if viewModel.viewState.isLoading {
+                    ProgressView()
+                        .zIndex(2)
+                } else if viewModel.viewState.linkedBanks?.isEmpty == false {
```

Comment:
This can be `} else if let linkedBanks = viewModel.viewState.linkedBanks, linkedBanks.isEmpty == false {`